common/src/lib.rs

```diff
@@ -52,6 +61,19 @@ pub const GAS_VALUE_PREFIX: &[u8] = b"g::gas_tree";
 
 pub type ExitCode = i32;
 
+#[derive(Clone, Debug, Decode, Encode, PartialEq, TypeInfo)]
+pub struct Dispatch {
+    pub kind: DispatchKind,
+    pub message: Message,
+}
+
+impl Dispatch {
```

Comment:
If I'm not mistaken this struct is not used there

Reply:
Now we store to the storage not just message, but a message with an entry-point, which is `Dispatch`.

`Dispatch` in `common` is the same as in `core`, except for message type. I will write down release notes